src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -153,7 +153,7 @@ export class HuiThermostatCard extends LitElement implements LovelaceCard {
         <g>
           <text text-anchor="middle" class="set-value">
             ${
-              !this._setTemp
+              this._setTemp === undefined
```

Comment:
```suggestion
              this._setTemp === undefined || this._setTemp === null
```
Better ---> 
<https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing>